+nigeLab/@Block/matchChannelID.m

```diff
@@ -47,7 +47,7 @@
    C = blockObj(iBk).ChannelID; % parse the channel IDs
    for iCh = 1:N % For each channel
       % Match the channel ID to the correct row (if it exists)
-      tmp = find(ismember(masterID,C(iCh,:),'rows'),1,'first');
+      tmp = find(ismember(C,masterID(iCh,:),'rows'),1,'first');
```

Comment:
@Nabarb How confident are you that this is correct? We're assigning `tmp` to `masterIdx` array for a given channel and block. So, shouldn't it be the corresponding index "into" the master, which will be accessed by some combination of indexing variables that indicate the current channel and block (elsewhere)?

Reply:
You are right and that is what it's doing, I believe. it looks for the index of the ID inside each block and logs the info into tmp, which is exactly the crossreferencing indexing variable you're talking about.
ie the channel with ID 8 is in position 8 in master but in position 7 in the third block, let's say. masterIdx(8,3) will be 7, and will be nan where the third block has the channel missing. Afterwards we will use masterIdx(iCh) when looking for the channel iCh in all blocks.

Reply:
OK as long as we are keeping that part consistent everywhere. Can we make a documentation of how "master" vs "child" Channel ID should be indexed, somewhere? Maybe it already exists...